CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 ## Testing and Infrastructure Changes:
 
 - Removed heavily-mocked tests related to comet_ml, wandb, and mlflow as they added a significant amount of test dependencies that prevented installation of test dependencies on Windows environemnts. By [@abidlabs](https://github.com/abidlabs) in [PR 3608](https://github.com/gradio-app/gradio/pull/3608)
+- Added Windows continuous integration, by [@space-nuko](https://github.com/space-nuko) in [PR 3628](https://github.com/gradio-app/gradio/pull/3628)
```

Comment:
Internal changes don't require a changelog entry.

@abidlabs @freddyaboulton We should remove this section from the changelog. Its adds noise for people who are interested in learning about bug fixes and new features. We can keep track of internal changes elsewhere if we want to but we should separate them from the changleog which is for users and should only contain changes with have a user-impact.

Reply:
Only downside is that if a community member makes an internal change (modifying a guide), they would not get credited

Reply:
That's true. I'll leave things as is in this PR, we can figure out a policy in a separate PR